sim/core/bulksim.go

```diff
@@ -495,23 +495,8 @@ func isValidEquipment(equipment *proto.EquipmentSpec) bool {
 		return false
 	}
 
-	// Validate trinkets/rings for duplicate IDs
-	if equipment.Items[proto.ItemSlot_ItemSlotFinger1].Id == equipment.Items[proto.ItemSlot_ItemSlotFinger2].Id && equipment.Items[proto.ItemSlot_ItemSlotFinger1].Id != 0 {
```

Comment:
There's no need to do this, as the FE UI doesn't allow for unique-equipped duplicates, and this would return as not valid if there's items that are not unique-equipped.